packages/react-components/src/components/Picker/Picker.spec.tsx

```diff
@@ -67,18 +76,56 @@ describe('<Picker> component', () => {
 
     userEvent.click(getByText('Select option'));
     userEvent.click(getByText('Option three'));
-    expect(onSelect).toHaveBeenCalledWith({
-      key: 'three',
-      name: 'Option three',
-    });
+    expect(onSelect).toHaveBeenCalledWith([
+      {
+        key: 'three',
+        name: 'Option three',
+      },
+    ]);
     userEvent.click(getByTestId('picker-trigger__clear-icon'));
     expect(onSelect).toHaveBeenCalledWith(null);
   });
 
+  it('should call onSelect with all correct elements in multiselect mode if "Select all" option is chosen', () => {
```

Comment:
I'm missing the test which allow to check the same thing but picking options one by one